wiki/Lua-Hyperspace-Module.md

```diff
@@ -1069,3 +1069,44 @@ local _, canMove = crew.extend:CalculateStat(Hyperspace.CrewStat.CAN_MOVE)
 - `std::vector<std::string>` `.hitShieldSounds`
 - `std::vector<std::string>` `.missSounds`
 - `string` `.image`
+
+## CommandConsole
+The members held by this class determine how commands run in the console behave.
+
+### Methods
+- `bool :RunCommand(CommandGui *commandGui, const std::string& command)`
+   - Used for running a Hyperspace command.
+- `CommandConsole .GetInstance()`
+   - Returns the main instance of `CommandConsole`. Always use this to access any members belonging to this class.
+
+### Fields
+- `bool` `.enabled`
+   - Whether or not the Command Console is enabled or not, this is set in Hyperspace.xml.
```

Comment:
You have a redundant "or not" in this sentence, one of them needs to be removed.